src/codegen/solana_accounts/account_collection.rs

```diff
@@ -505,3 +452,79 @@ fn check_expression(expr: &Expression, data: &mut RecurseData) -> bool {
 
     true
 }
+
+fn transfer_accounts(
+    loc: &pt::Loc,
+    contract_no: usize,
+    function_no: usize,
+    data: &mut RecurseData,
+    program_id_present: bool,
+) {
+    let accounts_to_add = data.functions[function_no].solana_accounts.borrow().clone();
```

Comment:
Why are you using both refcell and also clone.

Reply:
The `solana_accounts` field is mutably borrowed in `data.add_accounts` to insert the accounts into the index map. I cannot borrow it here.